test/assignment7-buildroot/script-helpers

```diff
@@ -0,0 +1,639 @@
+#!/bin/sh
```

Comment:
@akshita-bhasin rather than copy this entire script, can we symlink back to the assignment 4 script and just add in the assignment 7 test portion?

Reply:
I thought I did that, thanks for noticing! Will fix it